### DISCUSSION TITLE
Java Melody and iBatis, database configuration

### DISCUSSION THREAD

**Discussion by nedad** [javamelody/javamelody#985] [Q&A]
Hello,

I've add Java Melody in pom.xml
```
<dependency>
	<groupId>net.bull.javamelody</groupId>
	<artifactId>javamelody-core</artifactId>
	<version>1.86.0</version>
</dependency>
<dependency>
	<groupId>org.jrobin</groupId>
	<artifactId>jrobin</artifactId>
	<version>1.5.9</version>
</dependency>
```
And in web.xml
```
<filter>
			<filter-name>javamelody</filter-name>
			<filter-class>net.bull.javamelody.MonitoringFilter</filter-class>
	</filter>
	<filter-mapping>
		<filter-name>javamelody</filter-name>
		<url-pattern>/*</url-pattern>
		<dispatcher>REQUEST</dispatcher>
	</filter-mapping>
	<listener>
		<listener-class>net.bull.javamelody.SessionListener</listener-class>
	</listener>
```
And it works, but I don't see any database statistics. I've a Spring project that has ibatis for persistence layer and Oracle database. How should look like the configuration for Java Melody for ibatis? Has Java Melody debugger or logger, so I can see what is not configured correctly. And how to use debugger or logger for Java Melody.

Thank you in advance.

Regards,
Neda

**Comment by evernat** [comment]
Hello Neda and welcome,
See https://github.com/javamelody/javamelody/wiki/UserGuide#7-jdbc

(I suggest to use a jdbc datasource from JNDI, with a JNDI name starting with `jdbc/`)

**Comment by nedad** [comment]
Hello and thank you!

SQL statistics are still not there.
When I look the log. I see:
`No annotated classes found for specified class/package [classpath:net/bull/javamelody/monitoring-spring.xml]`
What does it mean exactly?
Is my bean correct configured?
`<bean id="jndiDataSource" class="org.springframework.jndi.JndiObjectFactoryBean">
			<property name="jndiName" value="${jdbc.name}" />
		</bean>`
And ${jdbc.name} is jdbc/XXX
Regards,
Neda
